apps/meteor/app/api/server/v1/im.ts

```diff
@@ -123,18 +123,35 @@ API.v1.addRoute(
 			if (!roomId) {
 				throw new Meteor.Error('error-room-param-not-provided', 'Body param "roomId" is required');
 			}
-			const canAccess = await canAccessRoomIdAsync(roomId, this.userId);
-			if (!canAccess) {
-				return API.v1.unauthorized();
+
+			let rid: IRoom['_id'] = roomId;
+			let subscription: ISubscription | null = null;
```

Comment:
```suggestion
			let subscription;
```
We only need to store the subscription here, and there's no need to specify the type since it can be infered from the returns later.
Generally, we should avoid using `let`. But I think that's a good use case for it.